README.md

```diff
@@ -108,9 +108,9 @@ Implement `<RocketsList>` component with the following specifications:
 - While 'missions' are being fetched from the API, the component renders `"Loading..."` on the screen.
 - If no 'missions' are obtained from the custom hook, the component renders `"No data"` on the screen.
 
-**Example:**
+**Expected outcome:**
 
-Considering we render `RocketList` like the following:
+Please make sure to set the `filterParams` to match the example below:
```

Comment:
I'm not sure if these two changes are necessary. Please advice whether you find it better this way or shall I revert.